comeet/UI/MainMenu/MainMenuViewController.swift

```diff
@@ -30,10 +36,60 @@ class MainMenuViewController: BaseViewController {
     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
         guard let identifier = segue.identifier,
             let viewModel = viewModel else {
+                print("")
                 return
         }
         Router.prepare(identifier: identifier, destination: segue.destination, sourceViewModel: viewModel)
     }
+    
+    
+    
+    @IBAction func openAgenda(_ sender: Any) {
+        
+        let viewController = SSCalendarDailyViewController(events: generateEvents())!
+        viewController.day = SSDayNode(date: Date())
+        let navigationController = self.navigationController!
+        navigationController.navigationBar.isTranslucent = false
+        navigationController.pushViewController(viewController, animated: true);
+    }
+    
+    
+    
+    
+    
+    //MARK: - SSCalendar setup
+    fileprivate func generateEvents() -> [SSEvent] {
```

Comment:
This function even if it's mocked should be in a MyAgendaMonthViewModel.swift class since it has nothing to do with the MainMenuViewController.swift.